scripts/bootstrap

```diff
@@ -54,6 +54,9 @@ function pull_sbtopts() {
     sort app-backend/.sbtopts | uniq > cleaned_sbtopts
     mv cleaned_sbtopts app-backend/.sbtopts
 
+    git config --local filter.gitignore.clean "sed '/#gitignore$/'d"
```

Comment:
With these Git configurations being made within the context of the virtual machine (where `bootstrap` gets executed by default), is there a way for them to be propagated back to the virtual machine host?

For example, if I'm using SourceTree on the host, how do these configuration changes make it back to the `.git` directory on my workstation given that the Vagrant `rsync` shared folder mechanism is one-way?

Reply:
Yeah, I have to rethink the approach here.

Reply:
@hectcastro This PR is ready for another pass. I pulled the execution of `./scripts/bootstrap` that pulls down `.sbtopts` out of the VM, and into `./scripts/setup`.